style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400;1,600&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html,
+body {
+  background-color: #fff;
+  color: #000;
+  font-family: "Poppins", sans-serif;
+  width: 400px;
+  margin: auto;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly make your logo color the same color as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205439593-84ebe960-f037-43eb-a9b6-4ef07906a8b4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205439585-337a8b84-c4c1-4062-976d-30afdf64367c.png)